.github/workflow-template/jobs/compute-node-test.yml

```diff
@@ -54,11 +54,16 @@ jobs:
       - name: Run rust doc check
         run: |
           cargo test --doc
-      
+
       - name: Run rust test with coverage
         run: |
           cargo llvm-cov nextest --lcov --output-path lcov.info -- --no-fail-fast
 
+      - name: Run deterministic test
+        continue-on-error: true
+        run: |
+          RUSTFLAGS="--cfg madsim" cargo test --target-dir target/sim --no-fail-fast -p risingwave_batch -p risingwave_stream -p risingwave_storage -p risingwave_connector -p risingwave_source -p risingwave_compute
```

Comment:
I guess this would definitely introduce a full rebuild. What about waiting for the new CI service and make it a separate workflow?

Reply:
`--target-dir target/sim` makes it output to a new directory and does not overwrite the original.
It takes 3 more minutes so maybe acceptable?